com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/Geometry/PositionNode.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using UnityEditor.ShaderGraph.GraphDelta;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class PositionNode : IStandardNode
+    {
+        static string Name = "Position";
+        static int Version = 1;
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Version,
+            Name,
+            "Out = Position;",
+            new ParameterDescriptor("Out", TYPE.Vec3, GraphType.Usage.Out),
+            new ParameterDescriptor("Position", TYPE.Vec3, GraphType.Usage.Static, defaultValue: REF.WorldSpace_Position)
+        );
+
+        public static NodeUIDescriptor NodeUIDescriptor => new(
+            Version,
+            Name,
+            tooltip: "Gets the location of the point in object, view, world, or tangent space.",
+            categories: new string[2] { "Input", "Geometry" },
+            synonyms: new string[1] { "location" },
+            parameters: new ParameterUIDescriptor[2] {
+                new ParameterUIDescriptor(
+                    name: "Position",
+                    options: REF.OptionList.Positions
```

Comment:
I guess this will auto-convert to a drop-down in the future?

Reply:
That's right.  None of the UI elements for the referables have been created yet.  Brett is working on them now.